### PR TITLE
TSK-976: Set up eslint

### PR DESCRIPTION
**Pull request by BVier**

### CODE REVIEW

File: web/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "project": "tsconfig.json",
+  },
+  "plugins": [
+        "@typescript-eslint",
+        "@typescript-eslint/tslint"
+  ],
+  "rules": {
+    "arrow-parens": ["off", "as-needed"],
```
Comment (mustaphazorgati) [inline web/.eslintrc.js]: where did you get the rules from? isn't the airbnb base good enough? Or is this just to deactivate the rules for now?

File: web/package.json [hunk 2]
```diff
@@ -10,11 +10,12 @@
     "build:prod-silent": "ng build --prod=true --no-progress",
     "test": "ng test --karma-config karma.conf.js --watch=false --browsers Firefox",
     "test:watch": "ng test --karma-config karma.conf.js --browsers Chrome",
-    "lint": "ng lint"
+    "lint": "eslint src/app/**/*.ts"
   },
   "private": true,
   "dependencies": {
     "@angular/animations": "8.2.11",
+    "@angular/core": "8.2.11",
```
Comment (mustaphazorgati) [inline web/package.json]: please use ^8.2.11

File: web/package.json [hunk 3]
```diff
@@ -64,7 +71,7 @@
     "protractor": "5.4.0",
     "ts-mockito": "2.3.0",
     "ts-node": "4.1.0",
-    "tslint": "5.9.1",
+    "tslint": "^5.9.1",
```
Comment (mustaphazorgati) [inline web/package.json]: still necessary, even after removal?

File: web/.eslintrc.js (hunk 1 above)
Comment (BVier) [reply web/.eslintrc.js]: Must rules are just deactivated to make the build run until they are fixed.
Some rules didn't matched our format but could be configured to match it.
(e.g. indent is 2 not 4)

File: web/package.json (hunk 2 above)
Comment (BVier) [reply web/package.json]: ok.
Only for angular cor?

File: web/package.json (hunk 3 above)
Comment (BVier) [reply web/package.json]: It shouldn't be, but my console required it...
Maybe it's because of the 'ng lint' settings which I didn't change yet.

Comment (BVier) [reply web/package.json]: But pls try it without this dependency - maybe it's only a local problem of mine!

File: web/package.json (hunk 2 above)
Comment (BVier) [reply web/package.json]: ✔️

File: web/package.json (hunk 3 above)
Comment (BVier) [reply web/package.json]: I thought so, but I had problems when I linted with my cmd.
After the rebase it works without this dependecy so I think it's ok to remove it!

**Comment by BVier** [comment]
Sry, I have no idea why the build is failing...

**Comment by mustaphazorgati** [comment]
> Sry, I have no idea why the build is failing...
Probably node cache. Restarted build. Let's see

**Comment by mustaphazorgati** [comment]
Almost.. Malformed package-lock.json

**Comment by mustaphazorgati** [comment]
fount it: https://github.com/angular/angular-cli/issues/15493#issuecomment-529098088

File: web/package.json (hunk 3 above)
Comment (mustaphazorgati) [reply web/package.json]: It's necessary for the eslint-tslint plugin

**Pull request merged**